tests/test_plot.py

```diff
@@ -118,6 +118,15 @@ def test_roc_multi_add_to_multi(
     roc1 + roc2
 
 
+@image_comparison(baseline_images=["pca"])
+def test_pca():
+    weights = np.array([73,95,72,69,76,82,95,89,89,83,94,89,103]) 
+    heights = np.array([56,73,60,54,56,62,77,73,72,69,74,67,84,]) 
+    X = np.array((heights,weights)).transpose()
+
+    plot.pca(X,standard_scalar=True)
+
```

Comment:
Please add some test case, e.g., 
1. Different input type like pandas
2. Create a dummy dataset and then run pca on it
3. check that passing when `ax` is passed the plot is not creating a new axes